### DISCUSSION TITLE
How to include an Angular Project within .NET Aspire

### DISCUSSION THREAD

**Discussion by giscus** [timdeschryver/timdeschryver.dev#294] [Announcements]
# How to include an Angular Project within .NET Aspire

Learn how to add and configure your Angular application in a .NET Aspire Application Host.

https://timdeschryver.dev/blog/how-to-include-an-angular-project-within-net-aspire

**Comment by codepatroltech** [comment]
Hi,

I can't find these extension in the 8.0.0-preview.1.23557.2 for Aspire.Hosting.

Are these npm extensions released?

Thanks.

**Reply by timdeschryver** [reply]
For now you'll have to write the npm extensions manually.
But, it's currently being added to .NET Aspire (https://github.com/dotnet/aspire/pull/1033).

**Reply by codepatroltech** [reply]
Thank you, I did some digging and found that thread. 
I appreciate your writing in this article, very prompt.

**Reply by timdeschryver** [reply]
@codepatroltech ping, the article has been updated with the latest Aspire version that includes the npm extensions 😉

**Comment by geokaps** [comment]
Hi Tim, thanks for the timely article! One question though: I know this adds the Angular app to the Aspire.AppHost for purposes of development, but how do we address deploying the Angular app as part of the Aspire project to Azure Container Apps, which is Aspire's target deployment environment on Azure?

**Reply by timdeschryver** [reply]
Good question @geokaps , to be honest I don't know. That's something I haven't looked at for now, maybe that the documentation can help you https://learn.microsoft.com/en-us/dotnet/aspire/deployment/overview?WT.mc_id=DT-MVP-5004452

**Comment by Marcel0024** [comment]
also tip:

`WithServiceBinding` is now `WithHttpEndpoint`

**Reply by timdeschryver** [reply]
Thanks for bringing this up @Marcel0024, I'll update the article later this week!

**Comment by dluo-sig** [comment]
Does this kind of setup allow breakpoints to be hit in the Angular application?  I could not get it to work, but perhaps I'm doing something wrong.
